source/CardGames.py

```diff
@@ -322,6 +302,16 @@ def ask_card(self):
     self.player_asked = self.playerinfo[self.index(player_asked, self.playerinfo)][0]
 
     self.surrender_card()
+  def count_cards(self, deck):
+    # How many cards are in the deck assigned to "num_cards"
+    num_cards = len(deck)
+    # For each iteration the number of cards in the deck decrease by 1 card
```

Comment:
Thank you for resolving this issue, even when it wasn't your function. It is really important that it works for the game. I might change it to where it doesn't take a deck parameter so that we don't have to include it every time. But this function is very good. Thanks again!

Reply:
Thanks for your comment Keren!